pkg/config/templates/rancherd-15-logging.yaml

```diff
@@ -0,0 +1,50 @@
+resources:
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging
+    releaseName: rancher-logging
+    version: {{ .LoggingChartVersion }}
+    defaultNamespace: cattle-logging-system
+    repoName: harvester-charts
+    diff:
+      comparePatches:
+        - apiVersion: admissionregistration.k8s.io/v1
+          kind: ValidatingWebhookConfiguration
+          name: rancher-logging-admission
+          operations:
+            - { "op": "remove", "path": "/webhooks" }
```

Comment:
The `diff` part is a special patch for `rancher-monitoring`, `rancher-logging` does not need it.

Reply:
removed

Reply:
removed